schemas/tls.yml

```diff
@@ -0,0 +1,196 @@
+---
+- name: tls
+  title: TLS
+  group: 2
+  short: Fields describing a TLS connection.
+  description: >
+    Fields related to TLS activity.
+  type: group
+  fields:
+
+    - name: version.protocol
+      level: extended
+      type: keyword
+      description: Normalized lowercase protocol name parsed from original string.
+      example: "tls` or `ssl"
```

Comment:
ECS can only handle single examples in here. Please use either "tls" or "ssl", but not both, it'll render weird :-)

Reply:
Oh, okay. I checked the rendered asciidoc output and it actually looked good, but I kinda hacked it with the backticks